packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -58,16 +58,17 @@ BuildApp() {
   fi
 
   if [[ $CURRENT_ARCH != "x86_64" ]]; then
-    local interpreter_flag="--interpreter"
-    if [[ "$DART_EXPERIMENTAL_INTERPRETER" != "1" ]]; then
-      interpreter_flag=""
+    local aot_flags=""
+    if [[ "$FLUTTER_MODE" == "debug" ]]; then
+      aot_flags="--interpreter --debug"
+    else
+      aot_flags="--${FLUTTER_MODE}"
```

Comment:
Can you set a default for `FLUTTER_MODE` (presumably release) and then use the mode to set the default value of framework_path?  Also, please update the hello_services docs.